src/main.ts

```diff
@@ -134,6 +136,19 @@ const createWindow = async () => {
   w.on('closed', () => {
     mainWindow = null
   })
+  w.once('ready-to-show', () => {
+    const serve = serveStatic(`${process.cwd()}/node_modules/squoosh/build`, {
+      index: ['index.html'],
+    })
+    // Create server
+    const server = http.createServer(function onRequest(req, res) {
+      serve(req, res, () => {
+        console.log('Create server')
+      })
+    })
+    // Listen
+    server.listen(5100)
+  })
```

Comment:
I think the server is not necessary. You can request `file:///node_modules/squoosh/build/index.html` (or something like this) directly in the modal

Reply:
@br4in3x I have tried with this but it doesn't work, when you tried to add like that, I will be shown 404. Refer this https://github.com/GoogleChromeLabs/squoosh/issues/999